airflow/hooks/dbapi_hook.py

```diff
@@ -70,8 +70,11 @@ def get_uri(self):
         host = conn.host
         if conn.port is not None:
             host += ':{port}'.format(port=conn.port)
-        return '{conn.conn_type}://{login}{host}/{conn.schema}'.format(
+        uri = '{conn.conn_type}://{login}{host}'.format(
             conn=conn, login=login, host=host)
+        if conn.schema is not None:
+            uri += '/{conn.schema}'
```

Comment:
This line looks wrong. You're not using a format method here but you're using `{}` inside.
It should be `uri += conn.schema` instead.